runtime/vm/JFRConstantPoolTypes.hpp

```diff
@@ -32,6 +31,8 @@
 #include "vm_internal.h"
 #include "ut_j9vm.h"
 
+#include <cstring>
+
 #if defined(J9VM_OPT_JFR)
```

Comment:
Please move the include of `cstring` after all openj9 and omr includes (after `VMHelpers.hpp`).
Also, line 36 should be moved before the include of `j9hypervisor.h`.

```c
#include "j9cfg.h"

#if defined(J9VM_OPT_JFR)

#include "j9hypervisor.h"
#include "j9.h"
#include "omr.h"
#include "omrlinkedlist.h"
#include "vm_api.h"
#include "vm_internal.h"
#include "ut_j9vm.h"
#include "BufferWriter.hpp"
#include "JFRUtils.hpp"
#include "ObjectAccessBarrierAPI.hpp"
#include "VMHelpers.hpp"

#include <cstring>
```